scripts/setup_udev_rules.sh

```diff
@@ -34,13 +44,22 @@ if [ "$install" = true ]; then
         echo -e "\e[0m"
     fi
     sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/
-    sudo cp config/99-realsense-d4xx-mipi-dfu.rules /etc/udev/rules.d/
+
+    if [[ -n "${is_ipu6}" ]] || [[ -n "${is_tegra}" ]]; then
+        sudo cp config/99-realsense-d4xx-mipi-dfu.rules /etc/udev/rules.d/
+        sudo cp scripts/rs-enum.sh /usr/local/bin/rs-enum.sh
+        sudo cp scripts/rs_ipu6_d457_bind.sh /usr/local/bin/rs_ipu6_d457_bind.sh
```

Comment:
Why not copy only relevant script based on is_tegra/is_ipu6?

Reply:
?

Reply:
trying to keep similarity for debian package (udev-rules)